src/pep257.py

```diff
@@ -61,7 +61,7 @@ def next(obj, default=nothing):
 
 
 __version__ = '0.7.1-alpha'
-__all__ = ('check', 'collect')
+__all__ = ('check')
```

Comment:
bah, this is incorrect.  should be a tuple, as shown in https://github.com/GreenSteam/pep257/pull/155 by @lordmauve